src/modules/kpi-reporting/controller.js

```diff
@@ -0,0 +1,51 @@
+'user-strict';
+const repo = require('../../lib/repo/kpi-reporting');
+const Boom = require('@hapi/boom');
+
+const months = ['January', 'February', 'March', 'April', 'May', 'June',
+  'July', 'August', 'September', 'October', 'November', 'December'];
+
+const percentageChage = (index, data, row) => {
+  if (index < (data.length - 1)) {
+    return (row.total - data[(index + 1)].total) / (data[(index + 1)].total < 1 ? 1 : data[(index + 1)].total) * 100;
```

Comment:
I'm not sure we should be handling the /0 issue in this way.  I'm not sure it is mathematically correct.
I wonder if we should do along the lines of:
```
const numerator = row.total - data[(index + 1)].total;
const denominator = data[(index + 1)].total;
if(numerator === 0 && denominator === 0) {
  return 0;
}
const result = numerator/denominator;
return result.isNaN() ? '∞' : result * 100;
```

Reply:
done thanks